pkg/windows/constants.go

```diff
@@ -0,0 +1,13 @@
+// +build windows
+
+package windows
+
+import "path/filepath"
+
+func GetWindowsAgentDir() string {
+	return filepath.Join("c:\\", "var", "lib", "rancher", "rke2", "agent")
```

Comment:
Are we always using `c:\\`?

Reply:
Yeah, I think this should use data-dir, rather than hardcoded.